src/components/GettingStartedModal/StepIndicator.tsx

```diff
@@ -0,0 +1,8 @@
+import styled from "styled-components";
+import { theme } from "../../theme";
+/*
+ * Defines css for steps.
+ */
+export const StepIndicator = styled.h4`
```

Comment:
Is this used by multiple files? If not, maybe it can just live in the file that uses it.

Reply:
This has been moved to the only file that uses it, index.tsx.